lib/ansible/utils/module_docs_fragments/eos.py

```diff
@@ -43,25 +43,29 @@ class ModuleDocFragment(object):
         either the CLI login or the eAPI authentication depending on which
         transport is used.
     required: true
+    env_default: ANSIBLE_NET_USERNAME
```

Comment:
We'll need to change all this documentation before release of 2.1.0 (feature freeze on the 18th of April).  We can move it into the description or adjust the module formatter (but the table format with a short word might not be good for it since fallback is generic now... I guess description probably is the best place for it for now.)